### ISSUE TITLE
specifying proportion for splitting command

### ISSUE THREAD

**Issue by abwolf** [gchen98/macs#34]
Hi Gary,

I just wanted to clarify the role of the 'p' value when using -es in macs. In the documentation it describes it as the proportion of chromosomes that move from pop i to pop i+1. So if I have a command macs ... I 2 1 1 ...-es 0.001 1 0.25..., this means that 25% of the chromosomes from pop1 will be moved to pop3?

This is different from how -es functions in ms, where the 'p' value specifies the probability of a chromosome ending up in population i. i.e. -es 0.001 1 0.25 in ms means that a chromosome from pop1 has a 25% probability of remaining in pop1, and a 75% probability of being moved to pop3?

Thank you.

**Comment by gchen98** [comment]
yes it should be the reverse of ms based on the source code here:

                     for (NodePtrSet::iterator it=pCoalescedNodes->begin();
                     it!=pCoalescedNodes->end();++it){
                         NodePtr node = *it;
                         if (node->getPopulation()==iSourcePop &&
                             pRandNumGenerator->unifRV()<dProportion){
                             nodesToMigrate.push_back(node);
                         }
                     }

hence the higher the value of p the higher the chance of a migration.
